activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessRuntimeImpl.java

```diff
@@ -225,6 +229,9 @@ public ProcessInstance start(StartProcessPayload startProcessPayload) {
         if (!securityPoliciesManager.canWrite(processDefinition.getKey())) {
```

Comment:
some parts of code in this class seem to be repeated, evaluate refactoring

Reply:
I agree that the codes repeated. Refactoring here may require too much changes not in the scope of this issue... Maybe it will have a sense to make once a big refactoring of these classes

Reply:
why don't we create a private method in the same class or in a utility class?
it will make the next refactoring smaller

Reply:
I have made here some refactoring